src/RscChart.tsx

```diff
@@ -104,14 +107,20 @@ export const RscChart = forwardRef<ChartHandle, RscChartProps>(
 
 		const sanitizedChildren = sanitizeRscChartChildren(props.children);
 
-		// THE MAGIC, builds our spec
+		// when data changes, make sure that we are animating from zero (especially in the case where a popover was just
+		// opened and closed)
+		useEffect(() => {
+			setAnimateFromZero(true);
+		}, [data]);
```

Comment:
Correct me if this is wrong, but I thought we were only wanting to animate from zero if data was too dissimilar to the previous data.  Is this accounted for later on in the code?

Reply:
Correct, it is!